src/open_inwoner/accounts/models.py

```diff
@@ -359,6 +359,9 @@ class Meta:
             models.CheckConstraint(
                 check=~Q(phonenumber__exact="") | Q(phonenumber_alternative__exact=""),
                 name="phonenumber_alt_requires_phonenumber_primary",
+                violation_error_message=_(
+                    "Primary phone number is required for setting alternative phone number"
```

Comment:
If we keep the verbatim error message we shouldn't need a new translation:


```suggestion
                    "A primary phone number is required for setting an alternative phone number"
```